app/webpacker/components/PostsWidget.js

```diff
@@ -62,7 +62,7 @@ function PostsList({
             <Card.Meta>
               Posted by
               {' '}
-              {post.author ? post.author.name : 'Unknown'}
+              {post.author_name ? post.author_name : 'Unknown'}
```

Comment:
I think this can be written as `post.author_name ?? 'Unknown'`

Reply:
Done, thanks